integreat_cms/cms/models/abstract_content_model.py

```diff
@@ -268,7 +268,15 @@ def __repr__(self):
         :rtype: str
         """
         class_name = type(self).__name__
-        return f"<{class_name} (id: {self.id}, region: {self.region.slug}, slug: {self.best_translation.slug})>"
+        # Only log the slug if doing so does not require a database query
+        slug = (
+            self.best_translation.slug
+            if hasattr(self, "prefetched_translations")
+            else "<...>"
+        )
+        return (
+            f"<{class_name} (id: {self.id}, region: {self.region.slug}, slug: {slug})>"
```

Comment:
I am a bit confused why pylint does such aggressive line breaks. Why would it break up the format string, even though it got shortened?